packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -416,6 +416,27 @@ describe('expression', () => {
         expect(result.warnings).toHaveLength(0);
     });
 
+    it('forbids extra curly brackets on text nodes', () => {
+        // Note: the special case with curly brackets on text nodes is because of Text node traversal does not properly
+        // builds the expression: const tokenizedContent = rawText.split(EXPRESSION_RE);
+        // converting {{foo}} into {{foo}, and the later is what's passed into the expression parser.
+        const result = parseTemplate(`<template>{{foo}}</template>`);
+
+        expect(result.warnings[0]).toMatchObject({
+            level: DiagnosticLevel.Error,
+            message: `Invalid expression {{foo} - LWC1060: Template expression doesn't allow ObjectExpression`,
```

Comment:
Is this the correct error message in this case?

Reply:
you could say so